lti_emailer/requirements/base.txt

```diff
@@ -1,14 +1,21 @@
 Django==1.8.16
-psycopg2==2.6.2
-ims_lti_py==0.6
-requests==2.12.1
-redis==2.10.5
-django-redis-cache==1.7.1
-hiredis==0.2.0
-django-cached-authentication-middleware==0.2.1
 django-angular==0.7.16
+django-cached-authentication-middleware==0.2.1
+django-redis-cache==1.7.1
 flanker==0.4.38
+hiredis==0.2.0
+ims_lti_py==0.6
 ndg-httpsclient==0.4.2
+psycopg2==2.6.2
+redis==2.10.5
+requests==2.12.1
+
+# flanker==0.4.38 requires cryptography but hasn't been updated in a long time
+cryptography==1.6
+
+# ng-httpsclient==0.4.2 requires pyopenssl but has no version specified
```

Comment:
oops, I meant `ndg-httpsclient`, which is still a requirement (line 11). I will update the comment and merge -- if I've misinterpreted your concern, let me know!